### ISSUE TITLE
@:native on classes doesn't rename the output class

### ISSUE THREAD

**Issue by bary12** [nadako/hxtsdgen#19]
I expect
```hx
@:expose
@:native("YourClass")
class MyClass { ... }
```
to output `export class YourClass { ... }` but instead it outputs `export class MyClass { ... }`.
A workaround is currently to write
```hx
@:expose("YourClass")
@:native("YourClass")
class MyClass { ... }
```
but this should be unnecessary as the class is compiled to YourClass in the JS output anyway.

**Comment by elsassph** [comment]
You can omit `@:native` if you set `@:expose("YourClass")`, but you're right it should be picked up.

**Comment by bary12** [comment]
I think non-js targets ignore `@:expose` so I still need `@:native` when compiling to multiple targets.

**Comment by elsassph** [comment]
Agreed `@:native` has been overlooked - shouldn't be long to fix.

**Comment by elsassph** [comment]
@bary12 would you mind trying the `fix/19_native` branch?

**Comment by bary12** [comment]
> @bary12 would you mind trying the `fix/19_native` branch?

Well, if the tests are passing is there really a need for me to clone and figure out how to set up a development branch?

**Comment by elsassph** [comment]
It's always good to have an approval :)

I'll go ahead but just FYI it's quite easy to point to a repo:

```bash
# pull lib from git
haxelib git [project-name] [git-clone-path] [branch]
haxelib git hxtsdgen https://github.com/nadako/hxtsdgen.git fix/19_native

# update git branch
haxelib update hxtsdgen

# cancel git mode
haxelib install hxtsdgen

# verify lib location
haxelib path hxtsdgen
```
https://lib.haxe.org/documentation/using-haxelib/

**Comment by elsassph** [comment]
Released as `0.2.0`
